models/bpreg/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhubai/base:latest
+
+LABEL authors="bspai@bwh.harvard.edu,lnuernberg@bwh.harvard.edu"
+
+# # clone mhub implementation
+ARG MHUB_MODELS_REPO
+
```

Comment:
Remove line-break between ARG and RUN here (ideally this looks exactly the same across all MHub Dockerfiles), and remove extra `#` in comment ;)